qa/suites/fs/top/mount/fuse.yaml

```diff
@@ -0,0 +1,2 @@
+tasks:
+  - ceph-fuse: [client.0]
```

Comment:
why not symlink `.qa/cephfs/mount/fuse.yaml`?

Reply:
oh yeh, sure.